### ISSUE TITLE
8.0.0 tries to read the Gradle version but version parser doesn't handle gradle nightly versions

### ISSUE THREAD

**Issue by ZacSweers** [bugsnag/bugsnag-android-gradle-plugin#522]
### Describe the bug
```
Could not determine the dependencies of task ':app:assembleExternalRelease'.
> Could not create task ':app:bugsnagReleaseExternal-armeabi-v7a-releaseTask'.
   > <8.3-20230426222550+0000> does not match format (\d+)\.(\d+)(?:\.)?(\d*)(\.|-|\+)?([0-9A-Za-z-.]*)?
```

### Steps to reproduce
Try using a gradle nightly or the version above, see that bugsnag fails to parse it

### Environment
* Android Studio version: N/A
* Gradle version: `8.3-20230426222550+0000`
* Android Gradle Plugin (AGP) version: 8.0.0
* Bugsnag Android Gradle Plugin version: 8.0.0
* Bugsnag manifest section (if modified): N/A

**Comment by johnkiely1** [comment]
Hey @ZacSweers, we are looking into a solution for this now. We will let you know as soon as its released.

**Comment by johnkiely1** [comment]
Hey @ZacSweers, this should be now fixed in our latest version [v8.0.1](https://github.com/bugsnag/bugsnag-android-gradle-plugin/releases/tag/v8.0.1). Let us know if you have any issues.

**Comment by ZacSweers** [comment]
Thanks!
